src/ObjectUtils/include/ObjectUtils/ObjectFile.h

```diff
@@ -67,6 +67,14 @@ class ObjectFile : public SymbolsFile {
 
   [[nodiscard]] virtual bool IsElf() const = 0;
   [[nodiscard]] virtual bool IsCoff() const = 0;
+
+  static constexpr uint64_t kUnknownSymbolSize = std::numeric_limits<uint64_t>::max();
+
+  [[nodiscard]] static bool SymbolInfoLessByAddress(const orbit_grpc_protos::SymbolInfo& lhs,
+                                                    const orbit_grpc_protos::SymbolInfo& rhs);
+
+  static void DeduceDebugSymbolMissingSizes(
```

Comment:
Where is this going to be used? You mentioned PDB files but the corresponding classes don't inherit from `ObjectFile`, so why place this method here?

Reply:
Ah, I haven't noticed that there is ObjectFile and SymbolFile. I moved it up to SymbolFile which is the common super class of CoffFile and PdbFile.